humilis_push_processor/resources.yaml.j2

```diff
@@ -44,7 +44,7 @@ resources:
         Code:
           S3Bucket: "{{lambda_function.s3bucket}}"
           S3Key: "{{lambda_function.s3key}}"
-        Runtime: "python2.7"
+        Runtime: "python3.6"
```

Comment:
We should only use Python3, as I don't think anyone is using the push processor layer at this point, I think it's OK, to just change it here instead of making this a parameter in the downstream project.